persistence/persistence-api/src/main/java/org/cbioportal/persistence/SampleRepository.java

```diff
@@ -9,10 +9,10 @@
 
 public interface SampleRepository {
     @Cacheable(cacheNames = "GeneralRepositoryCache", condition = "@cacheEnabledConfig.getEnabled()")
-    List<Sample> getAllSamples(String keyword, String projection, Integer pageSize,
+    List<Sample> getAllSamples(String keyword, List<String> studyIds, String projection, Integer pageSize,
```

Comment:
If you want to, we can. I was following the naming convention as seen in `PatientController`, and `GeneController`. In both instances, the method name is `getAll...` despite the fact that it has a keyword parameter.  
I agree this is misleading, and I'm happy to change it, but maybe it would be better to do it in all the places.